catlin/pkg/validator/content_validator.go

```diff
@@ -54,6 +54,10 @@ func (v *ContentValidator) Validate() Result {
 		result.Error("%s is missing a mandatory annotation for minimum pipeline version(%q)", resName, consts.MinPipelineVersionAnnotation)
 	}
 
+	if _, ok := annotations[consts.CategoryAnnotation]; !ok {
+		result.Error("%s is missing a mandatory annotation for category(%q)", resName, consts.CategoryAnnotation)
+	}
```

Comment:
apart from this should we validate the category with the provided list of categories so that author of the PR on catalog repository doesn't enters a different category. WDYT?

Reply:
+1

Reply:
Yes right but I think that would come under a separate pr because the list needs to be added in Hub first